mitiq/zne/tests/test_inference.py

```diff
@@ -342,6 +347,18 @@ def test_richardson_extr(test_f: Callable[[float], float]):
     assert np.isclose(fac._opt_params[-1], zne_value)
 
 
+def test_runge_extr(test_f: Callable[[float], float]):
```

Comment:
```suggestion
@mark.parametrize("test_f", [f_lin, f_non_lin])
def test_runge_extr(test_f: Callable[[float], float]):
```
I think the decorator to loop over test functions is missing

Reply:
Yes! I am still learning how to test in python.